src/web/releases.rs

```diff
@@ -528,13 +564,11 @@ pub fn search_handler(req: &mut Request) -> IronResult<Response> {
                      OFFSET FLOOR(RANDOM() * 280) LIMIT 1",
                     &[]
                 ));
-                //                                        ~~~~~~^
-                // FIXME: This is a fast query but using a constant
-                //        There are currently 280 crates with docs and 100+
-                //        starts. This should be fine for a while.
-                let name: String = rows.get(0).get(0);
-                let version: String = rows.get(0).get(1);
-                let target_name: String = rows.get(0).get(2);
+                let row = rows.into_iter().next().unwrap(); // TODO: Is this really infallible?
```

Comment:
> Is this really infallible?

As long as there is a crate with >= 100 stars that succeeded to build. So it might fail locally but it will never fail in prod. We can redo the logic as part of fixing #789 but it's fine for now.